scopes/compositions/compositions/composition.section.tsx

```diff
@@ -12,7 +12,8 @@ export class CompositionsSection implements Section {
      * simulations ui extension.
      */
     private compositions: CompositionsUI,
-    private options: Options
+    private options: Options,
+    private emptyStateSlot: EmptyStateSlot
```

Comment:
move `emptyStateSlot` inside the options, like we do for options.menuBarWidgetSlot

Reply:
leaving this for now, as it just gives errors if I add it here